src/LSST/system/TelemetryCom.cpp

```diff
@@ -161,15 +161,15 @@ void TelemetryCom::_server() {
         LINFO("TelemetryCom::server() accepting new client");
         {
             auto handlerThrd =
-                    ServerConnectionHandler::Ptr(new ServerConnectionHandler(sock, _telemetryMap->copyMap()));
+            		ServerConnectionHandler::Ptr(new ServerConnectionHandler(sock, _telemetryMap));
```

Comment:
Is this code format correct? See the numbers of space just before the `ServerConnectionHandler::Ptr`.